lib/nrf_modem_lib/nrf_modem_lib_trace.c

```diff
@@ -36,6 +36,54 @@ K_SEM_DEFINE(trace_done_sem, 1, 1);
 static int trace_init(void);
 static int trace_deinit(void);
 
+static double min_bps = DOUBLE_MAX;
+static double max_bps;
+static double tot_bps;
+static int num_bps;
+
+#if CONFIG_NRF_MODEM_LIB_TRACE_LOG_BITRATE
+#define LOG_PERIOD_MSEC K_MSEC(CONFIG_NRF_MODEM_LIB_TRACE_LOG_BITRATE_PERIOD_MSEC)
```

Comment:
This is no longer msec.
```suggestion
#define LOG_BITRATE_PERIOD K_MSEC(CONFIG_NRF_MODEM_LIB_TRACE_LOG_BITRATE_PERIOD_MSEC)
```

Reply:
I don't understand this comment. Can you elaborate?

Reply:
Discussed offline, but for reference, it was just a suggestion to change the name of the define as it is not a value in milliseconds.